src/core/GUIDReferenceHandler.ttslua

```diff
@@ -99,7 +99,8 @@ local GuidReferences = {
     TokenSpawnTracker        = "e3ffc9",
     TourStarter              = "0e5aa8",
     Trash                    = "70b9f6",
-    VictoryDisplay           = "6ccd6d"
+    EffectWatcher            = "87e529",
```

Comment:
This table is alphabetically sorted 🤣 
Should insert it at a different spot or actually just rename it to `Watcher` and put it at the end.